sdk/core/azure-core/CHANGELOG.md

```diff
@@ -26,6 +29,7 @@
 - Added `Azure::Core::Operation<T>::GetRawResponse()`.
 - Added `Azure::Core::PackageVersion`.
 - Added support for logging to console when `AZURE_LOG_LEVEL` environment variable is set.
+- Added `Azure::Core::Context::GetDeadline()`.
```

Comment:
This is in the wrong release. Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/2069